packages/forms-web-app/src/controllers/examination/process-submission/utils/process.js

```diff
@@ -20,6 +20,8 @@ const handleProcessSubmission = async (session) => {
 			}
 		}
 		logger.info('Processing files complete');
+		examinationSession.submissionId = submissionId;
```

Comment:
You want the api call to /complete before setting the submission item.

it thats 2xx then continue otherwise throw. (Maybe a custom error / error log )

Reply:
Added the complete API call.  You may want to take a look at the structure, there is a new exception which will log.